doc/mgr/alerts.rst

```diff
@@ -0,0 +1,50 @@
+Alerts module
+=============
+
+The alerts module can send simple alert messages about cluster health
+via e-mail.  In the future, it will support other notification methods
```

Comment:
are we trying to implement a mini version of [aodh](https://docs.openstack.org/aodh/latest/) ?

Reply:
Just something super-simple so you can get email alerts when ceph (health) is unhappy, without any external monitoring infrastructure.  Something simple that works out-of-the-box.